apps/gitness/src/pages-v2/connectors/connectors-list-page.tsx

```diff
@@ -0,0 +1,6 @@
+import { ConnectorsListPageView } from '@harnessio/ui/views'
+
+// temp component for testing in standalone
```

Comment:
should remove this, check if we can use portal app to render it

Reply:
removed and moved to design preview app